lib/Textmaster/Model/Project.php

```
@@ -319,6 +320,42 @@ public function addDocuments(array $documents)
     /**
      * {@inheritdoc}
      */
+    public function getPotentialAuthors($status = null)
+    {
+        if (null === $this->getId()) {
+            throw new BadMethodCallException(
+                'The project must be saved before getting authors who can do it.'
+            );
+        }
+
+        $response = $this->getApi()->authors($this->getId())->all($status);
+
+        $nextPage = $response['next_page'];
+        $authors = array_map(function ($author) {
+            return $author['author_id'];
+        }, $response['my_authors']);
+
+        while (null !== $nextPage) {
+            $response = $this->getApi()->authors($this->getId())->setPage($response['page'] + 1)->all('my_textmaster');
+
+            $nextPage = $response['next_page'];
+            $authors = array_merge($authors, array_map(function ($author) {
+                return $author['author_id'];
+            }, $response['my_authors']));
+        }
+
+        if (count($authors) === 0) {
+            throw new ErrorException(
```

Not sure we should throw an exception